_data/banking.yml

```diff
@@ -753,11 +753,12 @@ websites:
 
     - name: San Francisco Fire Credit Union
       url: https://www.sffirecu.org
-      twitter: SFFireCU
-      facebook: sffirecu
       img: sffirecu.png
-      tfa: No
-      status: https://twitter.com/sffirecu/status/577532435228991488
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      phone: Yes
+      doc: https://www.sffirecu.org/access/online-and-mobile-banking/new-olb
```

Comment:
Is there a better link which tells customers how to actually enable 2fa?
Also your link only mentions email and sms, not phone calls. Is that accurate?